docs/azure/sdk/authentication/local-development-broker.md

```diff
@@ -0,0 +1,123 @@
+
+:::zone-end
+
+:::zone target="docs" pivot="os-macos"
+
+Authentication brokers aren't preinstalled on macOS, but they are available through apps developed by Microsoft, such as Company Portal. These apps are installed when a macOS device is enrolled in a company device fleet via a solution like Microsoft Intune. To learn more about Apple device set up with the Microsoft Identity Platform, refer to [Microsoft Enterprise SSO plug-in for Apple devices](/entra/identity-platform/apple-sso-plugin).
+
+:::zone-end
+
+:::zone target="docs" pivot="os-linux"
+
+The Linux operating system uses [Microsoft single sign-on for Linux](/entra/identity/devices/sso-linux) as its authentication broker.
+
+> [!NOTE]
+> Microsoft SSO for Linux authentication broker support is introduced with `Azure.Identity.Broker` version v1.3.0.
+
+:::zone-end
+
+## Configure the app for brokered authentication
+
+Complete the following steps to enable the application to authenticate through the broker flow:
+
+1. In the [Azure portal](https://portal.azure.com), navigate to **Microsoft Entra ID** and select **App registrations** on the left-hand menu.
+1. Select the registration for your app, then select **Authentication**.
+1. Add the redirect URI to your app registration via a platform configuration:
+    1. Under **Platform configurations**, select **+ Add a platform**.
+    1. Under **Configure platforms**, select the tile for your application type (platform) to configure its settings, such as **mobile and desktop applications**.
+    1. In **Custom redirect URIs**, enter the following redirect URI for your platform:
+
+        | Platform    | Redirect URI                                                                                                          |
+        |-------------|-----------------------------------------------------------------------------------------------------------------------|
+        | Windows 10+ | `ms-appx-web://Microsoft.AAD.BrokerPlugin/your_client_id`                                                             |
+        | macOS       | `msauth.com.msauth.unsignedapp://auth` for unsigned apps `msauth.BUNDLE_ID://auth` for signed apps                    |
+        | WSL         | `ms-appx-web://Microsoft.AAD.BrokerPlugin/your_client_id`                                                             |
+        | Linux       | `https://login.microsoftonline.com/common/oauth2/nativeclient`                                                        |
```

Comment:
Note: zone pivots can't exist in the middle of numbered lists, so I put this table here to avoid 3 duplicate includes for one URL change.